AGENTS.md

```diff
@@ -0,0 +1,215 @@
+- **Icons not found**: Run `pnpm run build:icons`
+- **Type errors**: Ensure all packages are built in correct order
+- **Stale cache**: Clear with `turbo clean` if needed
+
+## Documentation Site
+
+### Live Development
+- Start with `pnpm run start`
+- Documentation lives in `packages/docs/`
+- Uses Next.js with live code examples
+- Component examples are interactive and editable
+
+### Adding Component Documentation
+1. Include live code examples using the docs component system
+2. Document all props with descriptions and examples following existing patterns
+3. Include accessibility guidelines and best practices
+
+## AI Agent Best Practices
+
+### Task Management
+- Use TodoWrite tool for multi-step component development
+- Break down complex tasks (new components, major refactors)
+- Track progress through build, test, and documentation phases
+
+### Code Analysis Approach
+1. **Understand existing patterns**: Read similar components first
+2. **Check dependencies**: Examine imports and package.json files
+3. **Follow conventions**: Match existing code style and patterns
+4. **Test thoroughly**: Run full test suite after changes
+
+### Error Handling
+- Build errors: Check TypeScript types and imports
+- Test failures: Review snapshots and component behavior
+- Lint errors: Follow established code formatting rules
+- Type errors: Ensure proper prop interfaces and theme usage
+
+### File Operations
+- **Prefer editing** existing files over creating new ones
+- **Follow naming conventions** established in the codebase
+- **Use MultiEdit** for multiple changes in the same file
+- **Read files first** to understand context before editing
+
+## Specific to BigDesign
+
+### Theme System
+- All styling uses `@bigcommerce/big-design-theme`
+- Use theme tokens for colors, spacing, typography
+
+### Icon System
+- Icons are SVG components in `@bigcommerce/big-design-icons`
+- Build icons before other packages: `pnpm run build:icons`
+- Icons are used throughout components for consistency
+
+### Accessibility
+- All interactive components must be keyboard accessible
+- Include proper ARIA attributes and roles
+- Test with screen readers when possible
+- Follow WCAG 2.2 AA guidelines
+
+### Browser Support
+- Modern browsers (Chrome, Firefox, Safari, Edge)
+- React 18 and styled-components 5.x
+- TypeScript strict mode enabled
+
+## Emergency Procedures
+
+### Broken Build
+1. Check TypeScript errors first
+2. Ensure icons are built: `pnpm run build:icons`
+3. Clear cache: `turbo clean`
+4. Rebuild: `pnpm run build`
+
+### Test Failures
+1. Check if snapshots need updating
+2. Verify component changes don't break existing behavior
+3. Run tests in isolation to identify specific failures
+4. Update snapshots carefully after reviewing changes
+
+### Deployment Issues
+1. All packages must build successfully
+2. Tests must pass completely
+3. Documentation must build without errors
+4. Icons must be available for all dependent packages
```

Comment:
Maybe we can just remove this to clear up some tokens? The models should be able to figure most of this out based on the above context.

Reply:
Removed.